terminology-guides/spanish-terminology.md

```diff
@@ -4,57 +4,62 @@
-| Contributors | Contribuyentes o colaboradores | The term "contribuidores" is the literal translation but is not a common used word for the term. Its use depends on the translation and the linguistic structure. |
-| Custom decorations | Decoraciones personalizadas | |
-| Daemon | Daemon | |
+| Blockchain | Blockchain / Cadena de Bloques | It maintains the english word because of the technology term. "Cadena de bloques" can also be used, inside Bitcoin.org [it's defined as "Cadena de bloques"](https://bitcoin.org/es/vocabulario#cadena-de-bloques). |
+| Community Crowdfunding System (CCS) | Sistema de Financiación Colectiva de la Comunidad | This is the first translation. We might find different definitions or leave "crowdfunding" as is, e.g. Sistema de Crowdfunding de la Comunidad. Any suggestions please open an issue.  |
+| Contributors | Colaboradores / Contribuyentes | The term "contribuidores" is the literal translation but is not a common used word for the term. Its use depends on the translation and the linguistic structure. "Contribuyentes" is a word for those who are charged with taxes but can also be used as "contributor". It's also a literal translation for "contributors". |
+| Custom decorations | Decoraciones Personalizadas | |
 | Exploit | Vulnerabilidad / Comprometido | |
 | File | Archivo | This is a better known word between spanish native speakers. |
 | Flag | Marcador | A "flag" is a mark/bookmark. In some of the GUI translations the word "opción" (option) is used. Opened discussions in #799 and #827. |
-| Forum Funding System - FFS | Sistema de Financiamiento Colectivo - SFC | |
-| Hash | Hash | --- |
-| Hash Rate | Índice de hash o tasa de hash | [Bitcoin.org "hash rate"](https://bitcoin.org/en/vocabulary#hash-rate) measurement  on which the hashing power is measured. |
-| Hashing Power | Poder de procesamiento o Poder de Hash | |
+| Fork | Bifurcación | Adding "bifurcación" as the Spanish translation for "fork". In Monero there are two fork updates every ~6 months where the old chain is left behind and we follow the new updated protocol chain. |
+| Forum Funding System (FFS) | Sistema de Financiamiento Colectivo - SFC | Is now Community Crowdfunding System (CCS) but we will keep the term in any case we refer to the FFS. |
+| Hash Rate | Índice de Procesamiento / Razón de Procesamiento / Tasa de Procesamiento | [Bitcoin.org "hash rate"](https://bitcoin.org/en/vocabulary#hash-rate) measurement  on which the hashing power is measured. |
+| Hashing Power | Poder de Procesamiento / Poder de Hash | |
 | Keys | Claves | "Llaves" was discussed but the Bitcoin community already uses "claves". |
-| Log categories | Categorias de registros | |
+| Log Categories | Categorias de Registros | |
```

Comment:
_Categorías_ must have a _tilde._ (not _categorias_).

Reply:
Thanks, done